flatliners/influxdbstorage.py

```diff
@@ -12,25 +12,26 @@ class InfluxdbStorage(BaseFlatliner):
     def __init__(self, influx_dsn):
         super().__init__()
         self.influx_dsn = influx_dsn
-        self.client = InfluxDBClient.from_dsn(self.influx_dsn, timeout=5)
+        self.client = InfluxDBClient.from_dsn(self.influx_dsn, timeout=15)
         self.buffer_list = []
         self.buffer_size = 5000
         _LOGGER.info("InfluxDB connection initialized.")
 
     def on_next(self, x):
         """ update l2 distance between cluster vector and baseline vector
         """
+        x.resource_deltas["weirdness_score"] = x.weirdness_score
         self.buffer_list.append({
             "measurement": "clusterdata",
             "tags": {
                 "clusterID": x.cluster,
                 "clusterVersion": x.version
             },
             "time": datetime.utcfromtimestamp(x.std_dev_timestamp).strftime('%Y-%m-%dT%H:%M:%SZ'),
-            "fields": {
-                "weirdness_score": x.weirdness_score
+            "fields": x.resource_deltas # {
```

Comment:
You probably want to put the delta in it's own field and use tags to indicate the resource type. This way you can group by tags. see http://docs.grafana.org/features/datasources/influxdb/
And use a new measurement.

```
measurment: resource_deltas
tags:  cluster_id, resource_type, cluster_version
fields: resource_delta
```